hecuba_core/src/SpaceFillingCurve.h

```diff
@@ -140,4 +147,19 @@ class ZorderCurveGenerator : public SpaceFillingCurve::PartitionGenerator {
 
 };
 
+
+class ZorderCurveGeneratorFiltered : public ZorderCurveGenerator {
+public:
+
+    ZorderCurveGeneratorFiltered(const ArrayMetadata *metas, void *data, std::list<std::vector<uint32_t> > &coord);
+
+    int32_t computeNextClusterId();
```

Comment:
override? :)

Reply:
yeah, i think is useful :) since c++11 this identifier can catch (using compile-time error) if the function being overridden  is not from the base class and virtual, so now and in the future can be helpful if other methods has to be implemented